modules/default/weather/weather.js

```diff
@@ -88,6 +89,8 @@ Module.register("weather", {
 		// Let the weather provider know we are starting.
 		this.weatherProvider.start();
 
+		this.config.feelsLikeWithDegree = this.translate("FEELS").indexOf("{DEGREE}") > -1;
```

Comment:
with DEGREE in all translations this would not be necessary either

Reply:
I would suggest to keep it as it is. As it is optional and very few languages will need DEGREE placeholder. If we add DEGREE placeholder it might break other modules who are using FEELS like resource string in their modules. New languages (Hindi and Gujarati) only have it now and if any third party is using it they need to correct it for this lang. only.